backend/src/openarchiefbeheer/conf/base.py

```diff
@@ -157,17 +157,11 @@
 
 ROOT_URLCONF = "openarchiefbeheer.urls"
 
-# List of callables that know how to import templates from various sources.
```

Comment:
What is this change for? Does it break any possible future Django template usage?

Reply:
By default, django uses the `"django.template.loaders.filesystem.Loader"`, and by setting `"APP_DIRS": True` it also uses `django.template.loaders.app_directories.Loader`. So the two are equivalent.

https://docs.djangoproject.com/en/5.1/ref/templates/api/#loader-types